securedrop/journalist_app/__init__.py

```diff
@@ -167,8 +146,9 @@ def setup_g() -> 'Optional[Response]':
             app.logger.error("Site logo not found.")
 
         if request.path.split('/')[1] == 'api':
-            pass  # We use the @token_required decorator for the API endpoints
-        else:  # We are not using the API
+            if request.endpoint not in _insecure_api_views and not logged_in():
```

Comment:
These if statements can be joined with an `and` I think

Reply:
I am not sure bout this one. If I join line 148 and 149 with an `and` then the app will always return a 403 weh using the API. But I agree that this dhould structured more clearly as it is crucial to understand authentication